### ISSUE TITLE
Ruffle Error on http://www.freeworldgroup.com/games8/gameindex/crazytracker_ruffle.htm

### ISSUE THREAD

**Issue by freeworldgroup** [ruffle-rs/ruffle#2280]
# Error Info
Error name: Error
Error message: panicked at 'already borrowed: BorrowMutError', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/gc-arena-0.2.0/src/gc_cell.rs:62:29
Error stack:
```
Error: panicked at 'already borrowed: BorrowMutError', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/gc-arena-0.2.0/src/gc_cell.rs:62:29
    at Object.Z (http://www.freeworldgroup.com/ruffle/core.ruffle.1e4046da431fecdb5396.js:1:20926)
    at __wbg_new_df6e6ab7a65c4c4d (http://www.freeworldgroup.com/ruffle/ruffle.js:1:38584)
    at http://www.freeworldgroup.com/ruffle/31a60fd287aee6ffe333.module.wasm:wasm-function[1463]:0x272b18
    at http://www.freeworldgroup.com/ruffle/31a60fd287aee6ffe333.module.wasm:wasm-function[1483]:0x2762ef
    at http://www.freeworldgroup.com/ruffle/31a60fd287aee6ffe333.module.wasm:wasm-function[4063]:0x33d17c
    at http://www.freeworldgroup.com/ruffle/31a60fd287aee6ffe333.module.wasm:wasm-function[3163]:0x31a677
    at http://www.freeworldgroup.com/ruffle/31a60fd287aee6ffe333.module.wasm:wasm-function[3695]:0x333977
    at http://www.freeworldgroup.com/ruffle/31a60fd287aee6ffe333.module.wasm:wasm-function[4403]:0x34274c
    at http://www.freeworldgroup.com/ruffle/31a60fd287aee6ffe333.module.wasm:wasm-function[4278]:0x340e0a
    at http://www.freeworldgroup.com/ruffle/31a60fd287aee6ffe333.module.wasm:wasm-function[4405]:0x3427b0
```

# Player Info
Allows script access: true

# Page Info
Page URL: http://www.freeworldgroup.com/games8/gameindex/crazytracker_ruffle.htm
SWF URL: http://www.freeworldgroup.com/games8/crazytracker/Crazytracker.swf

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2021-01-05
Channel: nightly
Built: 2021-01-05T00:48:08.020Z
Commit: de761f5b7c6753ad140018cbc38ba777201f739d

**Comment by Poopooracoocoo** [comment]
crash occurs when you hover over buttons

**Comment by Toad06** [comment]
Thanks for the report.

The panic happens when the cursor is over one of the buttons on the title screen, code is located in `\__Packages\page\Page_menu`.

This message is also logged many times when the game loads:
`[ERROR ruffle_core::avm1::activation] Unknown AVM1 opcode: Unknown { opcode: 252, data: [3, 0] }`

So all of this is possibly caused by obfuscation (cc. #458).

**Comment by Toad06** [comment]
Fixed by #3322.
